lib/mayaUsd/utils/traverseLayer.cpp

```diff
@@ -55,14 +55,14 @@ using AlreadyTraversedPaths = std::set<SdfPath>;
 
 void _traverseLayer(
     const SdfLayerHandle&           layer,
-    const SdfPath&                  path,
+    const SdfPath                   path,
```

Comment:
Why can't we pass by reference? Can pathsToCopy be Null?

Reply:
There was a problem with it becoming invalid during recursion for some reason. I did not investigate the fundamental reason.

Reply:
I just tried again to pass by reference and again it leads to failure. It seems related to temporary variables getting mangled during recursion. Given the obscure nature of USD SdfSpec and the templated code, I currently do not have a clear explanation of why this is so.

Reply:
Found the root cause, fixed it.

Reply:
Nice!